doc/ModulesDevelopment.md

```diff
@@ -410,6 +410,21 @@ __Live detector data__
 
 If the detector is ready and connected to the CRU(s), one can of course start the full data taking workflow, including the SubTimeFrameBuilder and the DPL processing and plug the QC onto it.
 
+## A more advanced example
+
+A more complete example is available. The config file is called [advanced.json](../Framework/advanced.json). The workflow is made of 3 sources, intermediate processing steps, 3 sinks and a Dispatcher connecting two QC tasks to a number of these steps and 2 checks:
```

Comment:
Please consider adding a sentence saying that the topology doesn't mean to represent any particular physics processing, it is just an example with multiple data processors.